config/settings-production.yml

```diff
@@ -51,16 +51,16 @@ buffy:
       add_as_assignee: false
     add_remove_checklist:
       - editor response
-        only: editors
-        template_file: editor_response.md
-        data_from_issue:
-          - reviewers-list
-          - author-handle
+          only: editors
```

Comment:
Looking at the heroku logs, i suspect the spacing here is breaking things. i'm not 100% sure but this maps to the spacing in the labels responders. 

<img width="1110" alt="Screenshot 2024-03-14 at 4 38 50 PM" src="https://github.com/pyOpenSci/buffy/assets/7649194/f6a17a74-81aa-4705-94b8-c826fc690f18">